public/views/userOverview.html

```diff
@@ -39,14 +39,17 @@
         </tr>
       </thead>
       <tbody>
-        <tr ng-repeat="row in rowCollection">
-          <td>{{row.firstName + ' ' + row.lastName}}</td>
-          <td>{{row.email}}</td>
-          <td>{{row.role}}</td>
-          <td>
+        <tr id="mydiv" ng-repeat="row in rowCollection">
+          <td id="{{row.isDeleted}}">{{row.firstName + ' ' + row.lastName}}</td>
```

Comment:
All of these `id="{{row.isDeleted}}"` should be a `bool` and be checked if `true` or `false` and then given a `class` of disabled instead of an `id`. `id`'s are unqiue and should not be used more than once.